ProcessOptimizer/tests/test_constraints.py

```diff
@@ -173,6 +174,52 @@ def test_Sum():
     for sample in samples:
         assert cons.validate_sample(sample)
 
+@pytest.mark.fast_test
+def test_SumEquals():
+    # Test that SumEquals type constraint can be initialized
+    SumEquals((1, 2, 3), 5)
+    SumEquals([3, 2, 1], -10.0)
+    
+    # You need to add at least two dimensions together
+    with raises(ValueError):
```

Comment:
Do you mean in a new test function ala `def test_SumEquals_something`? Because the other constraint types have all their tests bundled into one function in the same manner.

Reply:
Yeah, more or less one test per asset (not quite, but something like that), so that each test only tests one thing. 

I am not sure how much overhead it introduces, but I think it is easier to read, and it discourages unhealthy reuse of resources (not that there are any here, but in general, I know that I can have a tendency to do that).

Reply:
It adds a ton of overhead if we want to do this consistently, so I'm not going to do this now. If you look at the e.g. the test function test_Sum, it contains 9 different assert statements, test_Conditional contains 18 etc.